app/services/users.js

```diff
@@ -4,14 +4,20 @@ const logger = require('../logger');
 const { User } = require('../models');
 const jwt = require('./jwt');
 const config = require('../../config');
+const mailer = require('./mailer');
 
 const createUser = data => {
   logger.info('Create User: ', data);
 
-  return User.create(data).catch(error => {
-    logger.error(error);
-    throw errors.databaseError(error.message);
-  });
+  return User.create(data)
+    .then(user => {
+      mailer.sendMail(user);
```

Comment:
You're not awaiting the mail, why?

Reply:
There is an await inside the mailer function

Reply:
That's not enough, you should await here too, remember than await is only a syntax sugar for promises